### PR TITLE
refactor: remove LNGDBLSIZE from back end

### PR DESCRIPTION
**Pull request by WalterBright**
replace with `tysize[TYldouble]`

### CODE REVIEW

File: src/backend/cdef.h [hunk 1]
```diff
@@ -587,7 +574,7 @@ typedef int bool;
 #endif
 
 #if _WINDLL
-#define COPYRIGHT "Copyright © 2001 Digital Mars"
+#define COPYRIGHT "Copyright � 2001 Digital Mars"
```
Comment (yebblies) [inline src/backend/cdef.h]: ?

Comment (WalterBright) [inline src/backend/cdef.h]: Ya got me there. I don't know what's different.

Comment (yebblies) [inline src/backend/cdef.h]: https://github.com/D-Programming-Language/dmd/pull/5508

**Comment by yebblies** [comment]
Auto-merge toggled on

**Comment by yebblies** [comment]
Auto-merge toggled off

**Comment by yebblies** [comment]
Auto-merge toggled on

**Comment by schuetzm** [comment]
This introduced a regression:
https://issues.dlang.org/show_bug.cgi?id=16027
https://forum.dlang.org/post/tbismdhjzvytyxrihzil@forum.dlang.org

**Pull request merged**